python/ray/experimental/channel/hccl_group.py

```diff
@@ -0,0 +1,209 @@
+import logging
+import os
+from typing import Optional
+
+import torch
+import torch.distributed as dist
+import torch_npu  # The torch_npu for communicate
```

Comment:
does Ray install this package?

Reply:
No, but this will only be called if it's installed by checking NPU_TORCH_PACKAGE_AVAILABLE